<launch>

    <arg name="pkg_name" value="duckiebot_detection"/>

    <node  name="duckiebot_detection_node" pkg="$(arg pkg_name)" type="duckiebot_detection_node.py" output="screen"></node>
    <node  name="duckiebot_distance_node" pkg="$(arg pkg_name)" type="duckiebot_distance_node.py" output="screen" ></node>
    <!-- <node  name="lane_following_node" pkg="$(arg pkg_name)" type="lane_following_node.py"  output="screen"></node>  -->
    <group ns="$(arg veh)">
        <node name="lanefollow_node" pkg="$(arg pkg_name)" type="follower.py" output="screen" required="true">
            <param name="veh" value="$(arg veh)" />
        </node>
    </group>
</launch>
